### PR TITLE
Irminger 7 deploy csv updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR includes initial updates in preparation for Irminger 7 cruise
GI01SUMO_Deploy.csv
GI02HYPM_Deploy.csv
GI03FLMA_Deploy.csv
GI03FLMB_Deploy.csv

Added by Collin - 3 glider deploy csvs for Irm 7.

**note that there are needed updates/additions to Eng bulk load and Sensor bulk load - to be completed later.

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Irminger 7 AM Updates [DT-64]](https://trello.com/c/zA9n9aEG/64-irminger-7-am-updates-dt-64)

File: deployment/GI01SUMO_Deploy.csv [hunk 1]
```diff
@@ -301,3 +301,53 @@ AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-000
 AR35-05,,,,GI01SUMO-SBD12-05-WAVSSA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-WAVSSA-06301,59.945118,-39.57401,,0,2664,
 AR35-05,,,,GI01SUMO-SBD12-06-METBKA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-METLGR-00036,59.945118,-39.57401,,0,2664,
 AR35-05,,,,GI01SUMO-SBD12-08-FDCHPA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-00006,,CGINS-FDCHPA-64403,59.945118,-39.57401,,-6,2664,
+AR46,,,,GI01SUMO-RID16-00-DCLENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-NDCL16-00049,,,,12,,
+AR46,,,,GI01SUMO-RID16-01-OPTAAD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-OPTAAD-00257,,,,12,,
+AR46,,,,GI01SUMO-RID16-02-FLORTD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORTD-01217,,,,12,,
+AR46,,,,GI01SUMO-RID16-03-CTDBPF000,7,1,,,CGMGI-01SUMO-00007,,CGINS-CTDBPF-50001,,,,12,,
```
Comment (cdobs) [inline deployment/GI01SUMO_Deploy.csv:307]: Shouldn't this be 50142?

File: deployment/GI01SUMO_Deploy.csv [hunk 2]
```diff
@@ -301,3 +301,53 @@ AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-000
...
+AR46,,,,GI01SUMO-SBD11-06-METBKA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-METLGR-00013,,,,0,,
+AR46,,,,GI01SUMO-SBD11-08-NUTNRB000,7,1,,,CGMGI-01SUMO-00007,,CGINS-NUTNRB-01094,,,,1,,
+AR46,,,,GI01SUMO-SBD12-00-DCLENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BDCL12-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD12-01-OPTAAD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-OPTAAD-00209,,,,1,,
+AR46,,,,GI01SUMO-SBD12-02-FLORTD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORTD-01218,,,,1,,
+AR46,,,,GI01SUMO-SBD12-03-HYDGN0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-HYDGN0-20009,,,,0,,
+AR46,,,,GI01SUMO-SBD12-04-PCO2AA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2AA-35263,,,,0,,
+AR46,,,,GI01SUMO-SBD12-05-WAVSSA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-WAVSSA-05981,,,,0,,
+AR46,,,,GI01SUMO-SBD12-06-METBKA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-METLGR-00015,,,,0,,
+AR46,,,,GI01SUMO-SBD12-08-FDCHPA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-FDCHPA-43401,,,,-6,,
```
Comment (cdobs) [inline deployment/GI01SUMO_Deploy.csv:353]: Reviewed CUID changes and deployment SNs for D00007. Should the NSIF CTDBP SN be 50142? That's what it is in the config sheet. Also, per discussion in the cg-data-team channel the HYD sns would be switched to the vendor SN.

File: deployment/GI01SUMO_Deploy.csv [hunk 3]
```diff
@@ -301,3 +301,53 @@ AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-000
...
+AR46,,,,GI01SUMO-SBD11-01-MOPAK0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-MOPAK0-43258,,,,0,,
+AR46,,,,GI01SUMO-SBD11-03-HYDGN0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-HYDGN0-20010,,,,0,,
+AR46,,,,GI01SUMO-SBD11-04-DOSTAD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-DOSTAD-00211,,,,1,,
+AR46,,,,GI01SUMO-SBD11-05-SPKIRB000,7,1,,,CGMGI-01SUMO-00007,,CGINS-SPKIRB-00229,,,,-6,,
+AR46,,,,GI01SUMO-SBD11-06-METBKA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-METLGR-00013,,,,0,,
+AR46,,,,GI01SUMO-SBD11-08-NUTNRB000,7,1,,,CGMGI-01SUMO-00007,,CGINS-NUTNRB-01094,,,,1,,
+AR46,,,,GI01SUMO-SBD12-00-DCLENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BDCL12-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD12-01-OPTAAD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-OPTAAD-00209,,,,1,,
+AR46,,,,GI01SUMO-SBD12-02-FLORTD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORTD-01218,,,,1,,
+AR46,,,,GI01SUMO-SBD12-03-HYDGN0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-HYDGN0-20009,,,,0,,
```
Comment (cdobs) [inline deployment/GI01SUMO_Deploy.csv]: Per discussion in cg-data-team channel, this should be 42356

File: deployment/GI01SUMO_Deploy.csv [hunk 4]
```diff
@@ -301,3 +301,53 @@ AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-000
...
+AR46,,,,GI01SUMO-RII11-02-FLORDG033,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORDG-04034,,,,130,,
+AR46,,,,GI01SUMO-RII11-02-PCO2WC051,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2WC-C0103,,,,40,,
+AR46,,,,GI01SUMO-RII11-02-PCO2WC052,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2WC-C0104,,,,80,,
+AR46,,,,GI01SUMO-RII11-02-PCO2WC053,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2WC-C0106,,,,130,,
+AR46,,,,GI01SUMO-RII11-02-PHSENE041,7,1,,,CGMGI-01SUMO-00007,,CGINS-PHSENE-P0171,,,,20,,
+AR46,,,,GI01SUMO-RII11-02-PHSENE042,7,1,,,CGMGI-01SUMO-00007,,CGINS-PHSENE-P0176,,,,100,,
+AR46,,,,GI01SUMO-SBC11-00-CPMENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BCPM01-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD11-00-DCLENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BDCL11-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD11-01-MOPAK0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-MOPAK0-43258,,,,0,,
+AR46,,,,GI01SUMO-SBD11-03-HYDGN0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-HYDGN0-20010,,,,0,,
```
Comment (cdobs) [inline deployment/GI01SUMO_Deploy.csv]: Per discussion in cg-data-team channel, this should be 42529

File: deployment/GI02HYPM_Deploy.csv [hunk 5]
```diff
@@ -53,3 +53,12 @@ AR35-05,,,,GI02HYPM-WFP02-04-CTDPFL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-000
...
 AR35-05,,,,GI02HYPM-MPM01-02-ZPLSGA010,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGINS-ZPLSGA-55066,59.97287,-39.53082,,150,2664,downward-looking
+AR46,,,,GI02HYPM-RIM01-00-SIOENG000,7,1,,,CGMGI-02HYPM-00007,,CGCON-SIOCON-10118,,,,2570,,
+AR46,,,,GI02HYPM-RIM01-02-CTDMOG039,7,1,,,CGMGI-02HYPM-00007,,CGINS-CTDMOG-13621,,,,160,,
+AR46,,,,GI02HYPM-WFP02-00-WFPENG000,7,1,,,CGMGI-02HYPM-00007,CGVEH-GWFPGX-10401,CGCON-GWFPGX-10401,,,,2560,,
+AR46,,,,GI02HYPM-WFP02-01-FLORDL000,7,1,,,CGMGI-02HYPM-00007,CGVEH-GWFPGX-10401,CGINS-FLORDL-03280,,,,2560,,
+AR46,,,,GI02HYPM-WFP02-03-DOSTAL000,7,1,,,CGMGI-02HYPM-00007,CGVEH-GWFPGX-10401,CGINS-DOSTAL-01476,,,,2560,,
+AR46,,,,GI02HYPM-WFP02-04-CTDPFL000,7,1,,,CGMGI-02HYPM-00007,CGVEH-GWFPGX-10401,CGINS-CTDPFL-00131,,,,2560,,
+AR46,,,,GI02HYPM-WFP02-05-VEL3DL000,7,1,,,CGMGI-02HYPM-00007,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,,,,2560,,
+AR46,,,,GI02HYPM-MPM01-02-ZPLSGA009,7,1,,,CGMGI-02HYPM-00007,,CGINS-ZPLSGA-55109,,,,150,,upward-looking
+AR46,,,,GI02HYPM-MPM01-02-ZPLSGA010,7,1,,,CGMGI-02HYPM-00007,,CGINS-ZPLSGA-55111,,,,150,,downward-looking
```
Comment (cdobs) [inline deployment/GI02HYPM_Deploy.csv:64]: Looks good.

File: deployment/GI03FLMA_Deploy.csv [hunk 6]
```diff
@@ -155,3 +155,29 @@ AR35-05,,,,GI03FLMA-RIS01-00-SIOENG000,6,1,2019-08-07T13:34:00,,CGMGI-03FLMA-000
...
+AR46,,,,GI03FLMA-RIM01-02-CTDMOG046,7,1,,,CGMGI-03FLMA-00007,,CGINS-CTDMOG-13437,,,,250,,
+AR46,,,,GI03FLMA-RIM01-02-CTDMOG047,7,1,,,CGMGI-03FLMA-00007,,CGINS-CTDMOG-13438,,,,350,,
+AR46,,,,GI03FLMA-RIM01-02-CTDMOG048,7,1,,,CGMGI-03FLMA-00007,,CGINS-CTDMOG-13439,,,,500,,
+AR46,,,,GI03FLMA-RIM01-02-CTDMOH049,7,1,,,CGMGI-03FLMA-00007,,CGINS-CTDMOH-13450,,,,750,,
+AR46,,,,GI03FLMA-RIM01-02-CTDMOH050,7,1,,,CGMGI-03FLMA-00007,,CGINS-CTDMOH-13451,,,,1000,,
+AR46,,,,GI03FLMA-RIM01-02-CTDMOH051,7,1,,,CGMGI-03FLMA-00007,,CGINS-CTDMOH-13452,,,,1500,,
+AR46,,,,GI03FLMA-RIS01-00-SIOENG000,7,1,,,CGMGI-03FLMA-00007,,CGCON-SIOCON-13993,,,,30,,
+AR46,,,,GI03FLMA-RIS01-03-DOSTAD000,7,1,,,CGMGI-03FLMA-00007,,CGINS-DOSTAD-00375,,,,30,,
+AR46,,,,GI03FLMA-RIS01-04-PHSENF000,7,1,,,CGMGI-03FLMA-00007,,CGINS-PHSENF-P0081,,,,30,,
+AR46,,,,GI03FLMA-RIS01-05-FLORTD000,7,1,,,CGMGI-03FLMA-00007,,CGINS-FLORTD-01300,,,,30,,
```
Comment (cdobs) [inline deployment/GI03FLMA_Deploy.csv:183]: Reviewed SNs. Looks good.

File: deployment/GI03FLMB_Deploy.csv [hunk 7]
```diff
@@ -155,3 +155,29 @@ AR35-05,,,,GI03FLMB-RIS01-00-SIOENG000,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-000
...
+AR46,,,,GI03FLMB-RIM01-02-CTDMOG066,7,1,,,CGMGI-03FLMB-00007,,CGINS-CTDMOG-13616,,,,250,,
+AR46,,,,GI03FLMB-RIM01-02-CTDMOG067,7,1,,,CGMGI-03FLMB-00007,,CGINS-CTDMOG-13617,,,,350,,
+AR46,,,,GI03FLMB-RIM01-02-CTDMOG068,7,1,,,CGMGI-03FLMB-00007,,CGINS-CTDMOG-13620,,,,500,,
+AR46,,,,GI03FLMB-RIM01-02-CTDMOH069,7,1,,,CGMGI-03FLMB-00007,,CGINS-CTDMOH-13444,,,,750,,
+AR46,,,,GI03FLMB-RIM01-02-CTDMOH070,7,1,,,CGMGI-03FLMB-00007,,CGINS-CTDMOH-13445,,,,1000,,
+AR46,,,,GI03FLMB-RIM01-02-CTDMOH071,7,1,,,CGMGI-03FLMB-00007,,CGINS-CTDMOH-13446,,,,1500,,
+AR46,,,,GI03FLMB-RIS01-00-SIOENG000,7,1,,,CGMGI-03FLMB-00007,,CGCON-SIOCON-13990,,,,30,,
+AR46,,,,GI03FLMB-RIS01-03-DOSTAD000,7,1,,,CGMGI-03FLMB-00007,,CGINS-DOSTAD-00376,,,,30,,
+AR46,,,,GI03FLMB-RIS01-04-PHSENF000,7,1,,,CGMGI-03FLMB-00007,,CGINS-PHSENF-P0105,,,,30,,
+AR46,,,,GI03FLMB-RIS01-05-FLORTD000,7,1,,,CGMGI-03FLMB-00007,,CGINS-FLORTD-01301,,,,30,,
```
Comment (cdobs) [inline deployment/GI03FLMB_Deploy.csv:183]: Reviewed SNs. Looks good.

File: deployment/GI01SUMO_Deploy.csv [hunk 8]
```diff
@@ -301,3 +301,53 @@ AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-000
...
+AR46,,,,GI01SUMO-RII11-02-FLORDG033,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORDG-04034,,,,130,,
+AR46,,,,GI01SUMO-RII11-02-PCO2WC051,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2WC-C0103,,,,40,,
+AR46,,,,GI01SUMO-RII11-02-PCO2WC052,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2WC-C0104,,,,80,,
+AR46,,,,GI01SUMO-RII11-02-PCO2WC053,7,1,,,CGMGI-01SUMO-00007,,CGINS-PCO2WC-C0106,,,,130,,
+AR46,,,,GI01SUMO-RII11-02-PHSENE041,7,1,,,CGMGI-01SUMO-00007,,CGINS-PHSENE-P0171,,,,20,,
+AR46,,,,GI01SUMO-RII11-02-PHSENE042,7,1,,,CGMGI-01SUMO-00007,,CGINS-PHSENE-P0176,,,,100,,
+AR46,,,,GI01SUMO-SBC11-00-CPMENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BCPM01-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD11-00-DCLENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BDCL11-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD11-01-MOPAK0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-MOPAK0-43258,,,,0,,
+AR46,,,,GI01SUMO-SBD11-03-HYDGN0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-HYDGN0-42529,,,,0,,
```
Comment (cdobs) [inline deployment/GI01SUMO_Deploy.csv:341]: HYD SN update looks good.

File: deployment/GI01SUMO_Deploy.csv [hunk 9]
```diff
@@ -301,3 +301,53 @@ AR35-05,,,,GI01SUMO-SBD12-04-PCO2AA000,6,1,2019-08-05T15:29:00,,CGMGI-01SUMO-000
...
+AR46,,,,GI01SUMO-SBD11-04-DOSTAD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-DOSTAD-00211,,,,1,,
+AR46,,,,GI01SUMO-SBD11-05-SPKIRB000,7,1,,,CGMGI-01SUMO-00007,,CGINS-SPKIRB-00229,,,,-6,,
+AR46,,,,GI01SUMO-SBD11-06-METBKA000,7,1,,,CGMGI-01SUMO-00007,,CGINS-METLGR-00013,,,,0,,
+AR46,,,,GI01SUMO-SBD11-08-NUTNRB000,7,1,,,CGMGI-01SUMO-00007,,CGINS-NUTNRB-01094,,,,1,,
+AR46,,,,GI01SUMO-SBD12-00-DCLENG000,7,1,,,CGMGI-01SUMO-00007,,CGCON-BDCL12-50010,,,,0,,
+AR46,,,,GI01SUMO-SBD12-01-OPTAAD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-OPTAAD-00209,,,,1,,
+AR46,,,,GI01SUMO-SBD12-02-FLORTD000,7,1,,,CGMGI-01SUMO-00007,,CGINS-FLORTD-01218,,,,1,,
+AR46,,,,GI01SUMO-SBD12-03-HYDGN0000,7,1,,,CGMGI-01SUMO-00007,,CGINS-HYDGN0-42356,,,,0,,
```
Comment (cdobs) [inline deployment/GI01SUMO_Deploy.csv:349]: HYD SN update looks good.

File: deployment/GI01SUMO_Deploy.csv (hunk 2 above)
Comment (batryn) [reply deployment/GI01SUMO_Deploy.csv:353]: Looks good. I updated the CTDBP-F (NSIF) serial number to 50142

File: deployment/GI03FLMA_Deploy.csv (hunk 6 above)
Comment (batryn) [reply deployment/GI03FLMA_Deploy.csv:183]: Looks good

File: deployment/GI02HYPM_Deploy.csv (hunk 5 above)
Comment (batryn) [reply deployment/GI02HYPM_Deploy.csv:64]: Looks good

File: deployment/GI05MOAS-GL469_Deploy.csv [hunk 10]
```diff
@@ -1,5 +1,9 @@
...
+KN221-04,,KN221-04,,GI05MOAS-GL469-01-FLORDM000,1,1,2014-09-10T23:00:00,2014-09-18T23:32:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-FLORDM-03528,59.93373,-39.47383,,1000,1000,
+KN221-04,,KN221-04,,GI05MOAS-GL469-02-DOSTAM000,1,1,2014-09-10T23:00:00,2014-09-18T23:32:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-DOSTAM-00341,59.93373,-39.47383,,1000,1000,
+KN221-04,,KN221-04,,GI05MOAS-GL469-04-CTDGVM000,1,1,2014-09-10T23:00:00,2014-09-18T23:32:00,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-CTDGVM-09172,59.93373,-39.47383,,1000,1000,
+AR46,,,,GI05MOAS-GL469-00-ENG000000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGCON-GLDROO-00469,59,-39,,1000,1000,
+AR46,,,,GI05MOAS-GL469-01-FLORDM000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-FLORDM-03583,59,-39,,1000,1000,
+AR46,,,,GI05MOAS-GL469-02-DOSTAM000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-DOSTAM-00161,59,-39,,1000,1000,
+AR46,,,,GI05MOAS-GL469-04-CTDGVM000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-CTDGVM-09238,59,-39,,1000,1000,
```
Comment (batryn) [inline deployment/GI05MOAS-GL469_Deploy.csv:9]: Looks good

File: deployment/GI05MOAS-PG515_Deploy.csv [hunk 11]
```diff
@@ -5,4 +5,11 @@ AR35-05,,AR35-05,,GI05MOAS-PG515-02-DOSTAM000,3,1,2019-08-06T17:20:00,2019-08-06
...
\ No newline at end of file
+AR35-05,,AR35-05,,GI05MOAS-PG515-06-PARADM000,3,1,2019-08-06T17:20:00,2019-08-06T21:28:00,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-PARADM-50185,59.9162,-39.26579,,200,200,glider leaked
+AR46,,,,GI05MOAS-PG515-00-ENG000000,4,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGCON-GLDRGP-00515,59,-39,,200,200,
+AR46,,,,GI05MOAS-PG515-01-CTDGVM000,4,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-CTDGVM-09231,59,-39,,200,200,
+AR46,,,,GI05MOAS-PG515-02-DOSTAM000,4,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-DOSTAM-00310,59,-39,,200,200,
+AR46,,,,GI05MOAS-PG515-03-FLORTM000,4,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTM-03501,59,-39,,200,200,
+AR46,,,,GI05MOAS-PG515-04-FLORTO000,4,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-FLORTO-01239,59,-39,,200,200,
+AR46,,,,GI05MOAS-PG515-05-NUTNRM000,4,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-NUTNRM-00661,59,-39,,200,200,
+AR46,,,,GI05MOAS-PG515-06-PARADM000,4,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDRGP-00515,CGINS-PARADM-50185,59,-39,,200,200,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply deployment/GI05MOAS-PG515_Deploy.csv:15]: looks good

File: deployment/GI05MOAS-GL365_Deploy.csv [hunk 12]
```diff
@@ -0,0 +1,5 @@
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
+AR46,,,,GI05MOAS-GL365-00-ENG000000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGCON-GLDROO-00365,59,-39,,1000,1000,
+AR46,,,,GI05MOAS-GL365-01-FLORDM000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-FLORDM-03124,59,-39,,1000,1000,
+AR46,,,,GI05MOAS-GL365-02-DOSTAM000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-DOSTAM-00281,59,-39,,1000,1000,
+AR46,,,,GI05MOAS-GL365-04-CTDGVM000,3,1,2020-08-15T00:00:00,,CGVGI-05MOAS-00000,CGVEH-GLDROO-00365,CGINS-CTDGVM-09061,59,-39,,1000,1000,
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply deployment/GI05MOAS-GL365_Deploy.csv:5]: looks good

File: deployment/GI05MOAS-GL469_Deploy.csv (hunk 10 above)
Comment (rtravis-whoi) [reply deployment/GI05MOAS-GL469_Deploy.csv:9]: looks good

File: deployment/GI01SUMO_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [reply deployment/GI01SUMO_Deploy.csv:353]: reviewed JB update to CTDBP-F (NSIF) - looks good

**Pull request merged**